src/main/content/_i18n/en.yml

```diff
@@ -166,7 +157,6 @@ support:
   business_critical: business-critical
   issues: issues
   open_case_chat_phone: Open a case online, by chat, or by phone
-  find_out_more: Get paid support
```

Comment:
![image](https://github.com/OpenLiberty/openliberty.io/assets/6392944/81e4249f-4f49-4331-8cf0-861c520d06ec)
I think this one is still in use